README.md

```diff
@@ -1,2 +1,124 @@
-# oe_starter_content
-OpenEuropa starter content types
+# OpenEuropa oe_starter_content
+
+The OpenEuropa Starter Content provides a starting template for OpenEuropa content modules.
```

Comment:
This bit you need to update to match what this repo is offering, use this for inspiration https://github.com/openeuropa/oe_content/blob/2.x/README.md, but one remark, these wont be "corporate" content types

Reply:
I will do this.